test/utils/stat.test.js

```diff
@@ -396,6 +396,10 @@ describe('stat', () => {
     it('should format and return default `averageDailyDose` data', () => {
       const data = {
         totalInsulin: 80,
+        weight: {
+          value: 70,
+          units: 'kg'
```

Comment:
```suggestion
          value: 154,
          unit: 'lb'
```
Better to test another units too, if to test if the default value is working.